src/test/java/org/synyx/urlaubsverwaltung/sicknote/sicknote/SickNoteInteractionServiceImplTest.java

```diff
@@ -349,47 +349,4 @@ void submit() {
         verify(sickNoteMailService).sendSickNoteSubmittedNotificationToSickPerson(sickNote);
         verify(sickNoteMailService).sendSickNoteSubmittedNotificationToOfficeAndResponsibleManagement(sickNote);
     }
-
-    @Test
```

Comment:
@derTobsch why did you delete this complete test?

Reply:
weil das nun via application events gemacht werden sollte, glaube ich. Lass mal direkt schauen.

Reply:
Hat sich geklärt, Test ist wieder drin, nur der alte Kalenderintegrationsteil ist raus